synchronization/src/cyface/java/de/cyface/synchronization/CyfaceAuthenticator.java

```diff
@@ -277,11 +277,12 @@ public Bundle hasFeatures(AccountAuthenticatorResponse response, Account account
      * @throws ResponseParsingException When the http response could not be parsed.
      * @throws SynchronisationException When the new data output for the http connection failed to be created.
      * @throws UnauthorizedException If the credentials for the cyface server are wrong.
+     * @throws NetworkErrorException when the connection's input or error stream was null (e.g. connection disabled)
```

Comment:
this error lead to a crash when I enabled wifi and disabled it very fast as soon as it was connected. This captures it softly as all other sync errors

Reply:
That is probably the error Manuel told me about today.